sdk/monitor/Azure.Monitor.Query/tests.yml

```
@@ -4,5 +4,13 @@ extends:
     template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: monitor
+      CloudConfig:
+        Public:
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+        UsGov:
+          SubscriptionConfiguration: $(sub-config-gov-test-resources)
+        China:
+          SubscriptionConfiguration: $(sub-config-cn-test-resources)
+      SupportedClouds: 'Public,UsGov,China'
```

```suggestion
      SupportedClouds: 'Public,UsGov,China'
```

I don't think you'll need to override the cloud config here as these match the default values.